src/ioctls/system.rs

```diff
@@ -462,6 +504,33 @@ mod tests {
         assert_eq!(vm.run_size(), kvm.get_vcpu_mmap_size().unwrap());
     }
 
+    #[test]
+    #[cfg(any(target_arch = "aarch64"))]
+    fn test_create_vm_with_ipa_size() {
+        let kvm = Kvm::new().unwrap();
+        if kvm.check_extension(Cap::ArmVmIPASize) {
+            let vm = kvm.create_vm_with_ipa_size(40).unwrap();
```

Comment:
Are there other valid values that we can use? 40 seems to be the default.

Reply:
Same above :)